### DISCUSSION TITLE
Deseq2 foldchange comparison order

### DISCUSSION THREAD

**Discussion by gsk7757** [biostars/biostar-handbook-issues#295] [Q&A]
Hello, 
I am practicing RNA-seq analysis through the RNA-seq by example handbook and I have a question regarding fold change. Since the design.csv contains sample "bored 1,2,3 and excited 1,2,3" and condition "bored, excited". When Deseq2 was performed, gene "AAA-750000-UP-4 has log2foldchange of 2.2. Does this mean condition "bored" was upregulated 2.2 times compared to the condition "excited" or is it reverse? Could you please clarify the order of the foldchange?
Thank you,

**Comment by ialbert** [comment]
Customarily, when two conditions are listed A and B, and a foldchange is mentioned, it means the ratio of B/A,  second condition relative to the first condition

That is the most common convention. 

In this case, bored vs excited, a fold change of 4 would mean that ratio excited/bored = 4

**Reply by gsk7757** [reply]
Thank you so much for the clarification.
